test/primitives/base-element-test.js

```diff
@@ -84,11 +84,11 @@ describe('BaseElement', function() {
     });
 
     it('returns true when they are equal', function() {
-      expect(el.id.equals('foobar')).to.be.true;
```

Comment:
@pksunkara yes, this allows this sort of (very common) stuff:

```js
// Simple value access
console.log(el.id);

// Simple value assignment
el.id = 'abc123';

// You still have full access to the element the long way:
el.meta.get('id').attributes.set('encoding', 'utf-8');
```

That last example isn't realistic for the ID, but a language or encoding attribute might make sense for description or response body content in the API namespace.